### PR TITLE
CS-16095 Packages upgrade

### PR DESCRIPTION
**Pull request by donnyysmit**

### CODE REVIEW

**Review by ccoenraad — CHANGES_REQUESTED** [review]
Looks great overall!
Please also update `CHANGELOG.md` (it is currently outdated).
Ideally we should follow the format of <https://keepachangelog.com/en/1.0.0/>.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,26 +1,50 @@
 # Changelog
 
+## 3.2.7 (Februari 22, 2022)
+
+### Changed:
```
Comment (ccoenraad) [inline CHANGELOG.md:5]: @donnyysmit nitpicking: Please remove `:`

**Review by ccoenraad — APPROVED** [review]

**Review by MichielVanDoorn — CHANGES_REQUESTED** [review]
In the Jira ticket it says:
- Update the minor-version of the @ksyos-package if there are any changes.

Now the question can be whether packages upgrades can be considered changes?
In 3.2.7 also only a package upgrade happened, so I think the answer is yes.

**Comment by donnyysmit** [comment]
> In the Jira ticket it says:
> 
> * Update the minor-version of the @ksyos-package if there are any changes.
> 
> Now the question can be whether packages upgrades can be considered changes? In 3.2.7 also only a package upgrade happened, so I think the answer is yes.

Bumping the patch version while we have `^`  doesn't change anything though, because `^` makes sure you always get the latest of the version you're in

**Review by MichielVanDoorn — APPROVED** [review]
top

**Pull request merged**